redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -112,17 +121,36 @@ internal class ViewFlexContainer(
         }
       }
 
+    // Either OnScrollChangeListenerCompat or OnScrollChangeListener. Created lazily.
+    private var onScrollListener: Any? = null
+
     init {
       layoutParams = LayoutParams(WRAP_CONTENT, WRAP_CONTENT)
       updateViewHierarchy()
     }
 
+    fun attachOrDetachScrollListeners() {
+      val child = getChildAt(0)
+      if (child is NestedScrollView) {
+        val listener = (onScrollListener as? OnScrollChangeListenerCompat)
```

Comment:
```suggestion
        val listener = (onScrollListener as OnScrollChangeListenerCompat?)
```
This is a nit, but `as? Foo` is a cast that can fail and falls back to null. `as Foo?` is a cast to a nullable type.

Reply:
Fewer bytecodes, too! https://kotlin.godbolt.org/z/YKz7bfao4

Reply:
Neat!